cvat/apps/engine/models.py

```diff
@@ -202,6 +202,7 @@ class Task(models.Model):
     status = models.CharField(max_length=32, choices=StatusChoice.choices(),
         default=StatusChoice.ANNOTATION)
     data = models.ForeignKey(Data, on_delete=models.CASCADE, null=True, related_name="tasks")
+    dimension = models.CharField(max_length=2, default="2d")
```

Comment:
the serializer has max_length=64 parameter. Probably it should use choices parameter.

Reply:
I would suggest implement it via enums. Please, look how "status" field was implemented with choices argument.

Reply:
Made changes accordingly.